sdk/servicebus/service-bus/test/utils/managementUtils.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  QueueOptions,
+  TopicOptions,
+  SubscriptionOptions,
+  ServiceBusAtomManagementClient
+} from "../../src";
+
+import { EnvVarKeys, getEnvVars } from "./envVarUtils";
+
+export async function recreateQueue(queueName: string, parameters: QueueOptions): Promise<void> {
+  const env = getEnvVars();
+  const client = await new ServiceBusAtomManagementClient(
```

Comment:
We are creating the management client every time there is a request to recreate an entity. Can we not do this once and re-use the client?

Reply:
Updated.

Looks like previously we didn't have to create a new client and could have passed in existing client from tests (Or even better the tests can all use one client instance)
Can be addressed as part of general tests refactoring in near future